core/src/main/res/values/strings.xml

```diff
@@ -178,6 +178,10 @@
         <item quantity="one">%d episode deleted.</item>
```

Comment:
The old string can be deleted

Reply:
Did you want me to remove the resource for all languages?  I removed from the default strings.xml

Reply:
Oh, sorry. Please only remove from the default file. The others are updated by Transifex.

Reply:
If I don't remove the CI lint checker fails